### PR TITLE
Feature - update status banner

### PR DESCRIPTION
**Pull request by joefitter**
* Removed project specific code - this now lives in as-pages
* pass colour to wrapper
* render children if present
* render title if present

### CODE REVIEW

**Comment by lennym** [comment]
This will need to be a major version update.

**Comment by joefitter** [comment]
nah

**Comment by joefitter** [comment]
context: we weren't even using this before

**Comment by lennym** [comment]
😐

File: src/licence-status-banner/index.jsx [hunk 1]
```diff
@@ -69,13 +67,7 @@ class LicenceStatusBanner extends Component {
             this.renderDates()
           }
           {
-            isInactiveVersion && <Fragment>
-              <p><Snippet>invalidLicence.summary.ppl_active</Snippet></p>
-              <p><Link page="project.version.read" versionId={this.props.licence.granted.id} label={<Snippet>{'invalidLicence.view'}</Snippet>} /></p>
```
Comment (wheelsandcogs) [inline src/licence-status-banner/index.jsx:74]: Do we not need to link to the active version any more?

Comment (joefitter) [reply src/licence-status-banner/index.jsx:74]: https://github.com/UKHomeOffice/asl-pages/pull/691/files#diff-01987d26fec6f3e56dec2aa0ed6d17d2R22-R28

**Pull request merged**